__tests__/__utils__/services/api.ts

```diff
@@ -2,8 +2,9 @@ import { rest } from 'msw'
 
 import { getUuid } from './database'
 import { RestResolver, createUrlRegex } from './utils'
+import { Instance } from '../../../src/utils'
 
-export function givenApi(resolver: RestResolver) {
+export function givenApi(resolver: ApiRestResolver) {
```

Comment:
This PR has this change that the next https://github.com/serlo/cloudflare-worker/pull/454/files reverts?

Reply:
yep, this was first necessary to make eslint happy. However in the next PR I noticed, that `req.body` is deprecated and therefore the type information here aren't necessary any more...